integration-tests/script/copy_resources.sh

```diff
@@ -14,13 +14,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+echo "Copying integration test resources."
+
 set -e
 
 # setup client keystore
 ./docker/tls/generate-client-certs-and-keystores.sh
 rm -rf docker/client_tls
 cp -r client_tls docker/client_tls
 
+# install druid jars
+rm -rf $SHARED_DIR/docker
```

Comment:
CI failures look legit. 
We also need to run `mkdir -p $SHARED_DIR` so that $SHARED_DIR is present. It's probably already created on everyone's machine who has ever run IT tests :)

Reply:
```
Warning:
The JKS keystore uses a proprietary format. It is recommended to migrate to PKCS12 which is an industry standard format using "keytool -importkeystore -srckeystore intermediate_ca_client.jks -destkeystore intermediate_ca_client.jks -deststoretype pkcs12".
cp: cannot create directory '/home/travis/shared/docker': No such file or directory
```